src/Appwrite/Certificates/AdapterProvider.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Appwrite\Certificates;
+
+class AdapterProvider
```

Comment:
Our naming should be consistent with our patterns. Let's always prefer a single word name where possible and avoid redundant parts in names. AdapterProvider can just be Adapter or Provider.